scripts/workflows/release/rollback.cjs

```
@@ -22,159 +22,189 @@ function execGit(command, allowError = false) {
+        execGit(`git tag -d v${targetVersion}`);
+        execGit(`git push origin --delete v${targetVersion}`);
+      }
     }
+
+    console.log("");
+    console.log(`✅ Rollback of v${targetVersion} complete`);
   } catch (error) {
-    log("error", `❌ Rollback failed: ${error.message}`);
-    log("warn", "Manual recovery may be required. Check git status and verify main/develop branches.");
-    process.exit(1);
+    console.error(`❌ Rollback failed: ${error.message}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)